easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-cuda-8.0-pat-gpu-6.5.1.eb

```diff
@@ -0,0 +1,45 @@
+# contributed by Luca Marsella (CSCS)
+#
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+#
+name = 'GROMACS'
+version = "2016.3"
+cudaversion = '8.0'
+patversion = '651'
+versionsuffix = '-cuda-%s-pat-gpu-%s' % (cudaversion, patversion)
+
+homepage = 'http://www.gromacs.org'
+description = """GROMACS is a versatile package to perform molecular dynamics,
+ i.e. simulate the Newtonian equations of motion for systems with hundreds to millions of particles."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'usempi': 'True', 'openmp': 'True'}
+
+source_urls = ['ftp://ftp.gromacs.org/pub/gromacs/']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = ' -DCMAKE_C_FLAGS="-g -fPIC" -DCMAKE_CXX_FLAGS="-g -fPIC" '
+
+# It would be good to specify the full path (/opt/cray/pe/perftools/6.5.1/modulefiles/perftools-lite-gpu) as the module file does not contain the version in the name, but the module is not found that way.
+dependencies = [('perftools-lite-gpu', EXTERNAL_MODULE)]
```

Comment:
Does this work ?

```
dependencies = [('perftools-lite/6.5.1', EXTERNAL_MODULE)]
```

Reply:
No this does not work:

```
omlins@dom101:~/9_soft_install/production.git> eb --detect-loaded-modules=unload -r -f easybuild/easyconfigs/g/GROMACS/GROMACS-2016.3-CrayGNU-17.08-cuda-8.0-pat-gpu-6.5.1.eb
== temporary log file in case of crash /dev/shm/omlins/easybuild/stage/tmp/eb-Z3AgDW/easybuild-f4DwdT.log
== resolving dependencies ...
ERROR: Missing modules for one or more dependencies marked as external modules: ['perftools-lite-gpu/6.5.1']
```